cli/cli-client/src/main/java/org/infinispan/cli/commands/kubernetes/Install.java

```diff
@@ -0,0 +1,52 @@
+package org.infinispan.cli.commands.kubernetes;
+
+import java.io.ByteArrayInputStream;
+import java.nio.charset.StandardCharsets;
+
+import org.aesh.command.Command;
+import org.aesh.command.CommandDefinition;
+import org.aesh.command.CommandResult;
+import org.aesh.command.option.Option;
+import org.infinispan.cli.commands.CliCommand;
+import org.infinispan.cli.impl.ContextAwareCommandInvocation;
+import org.infinispan.cli.impl.KubernetesContextImpl;
+import org.infinispan.commons.util.TypedProperties;
+import org.kohsuke.MetaInfServices;
+
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.dsl.ParameterNamespaceListVisitFromServerGetDeleteRecreateWaitApplicable;
+
+/**
+ * @author Tristan Tarrant &lt;tristan@infinispan.org&gt;
+ * @since 12.0
+ **/
+@MetaInfServices(Command.class)
+@CommandDefinition(name = Install.CMD, description = "Installs the operator")
+public class Install extends CliCommand {
+   public static final String CMD = "install";
+
+   @Option(shortName = 'n', description = "Select the namespace", required = true)
+   String namespace;
+
+   @Option(shortName = 'h', hasValue = false, overrideRequired = true)
+   protected boolean help;
+
+   @Override
+   public boolean isHelp() {
+      return help;
+   }
+
+   @Override
+   public CommandResult exec(ContextAwareCommandInvocation invocation) {
+      KubernetesClient client = ((KubernetesContextImpl)invocation.getContext()).getKubernetesClient();
+      String crd = Kube.loadResourceAsString("operator/operator-install.yaml", new TypedProperties().setProperty("NAMESPACE", namespace));
+
+      ParameterNamespaceListVisitFromServerGetDeleteRecreateWaitApplicable<HasMetadata, Boolean> op = client.load(new ByteArrayInputStream(crd.getBytes(StandardCharsets.UTF_8)));
+      if (namespace != null) {
+         op.inNamespace(namespace);
+      }
+      op.createOrReplace();
```

Comment:
Install has following cases need to be handled
- The operator can be already installed with other or the same version. CRD and ClusterRoles are cluster wide resources and can be detected before install
- The installation account may have insufficient rights and cluster may remain some garbage resources after unsuccessful installation